addons/web/static/tests/core/py_js/py_date_tests.js

```diff
@@ -55,6 +57,22 @@ QUnit.module("py", {}, () => {
             assert.strictEqual(evaluateExpr(expr3), "2001-04-03 00:32:12");
         });
 
+        QUnit.test("to_utc", (assert) => {
+            const unpatchDate = mock.patchDate(2021, 8, 17, 16, 22, 0);
```

Comment:
we have a patchDate function in helpers/utils, can't you use it?

Reply:
also, hours and minutes aren't relevant here right? I would set them to 0 then, I got confused because of them

Reply:
> use

The patchDate function you mention modifies luxon.Settings. Not sure it does the job here.